web-client/src/views/TrialSessions/TrialSessionsTable.tsx

```diff
@@ -87,57 +87,82 @@ export const TrialSessionsTable = connect(
                   );
                 }
                 if (isTrialSessionRow(row)) {
+                  const additionalRow = row.sessionType === 'Special' && (
+                    <tr className="special-sessions-row">
+                      <td></td>
+                      <td></td>
+                      <td></td>
+                      <td colSpan={6}>
+                        <div className="special-session">
+                          <span>
```

Comment:
I think we can get rid of this outer span

Reply:
The format for bolding the leading text before the colon was the reason I had to do it this way. Do you have another way of accomplishing `Bolded text: regular text`?

Reply:
I could be missing something, but here's what it looks like for me without that outer span:

<img width="1443" alt="Screenshot 2024-11-05 at 10 51 48 AM" src="https://github.com/user-attachments/assets/c8f77a33-5eaa-49e5-a99e-d6f403b75e4e">

Reply:
I stand corrected!